cmd/internal/cli/pull.go

```diff
@@ -180,9 +185,9 @@ func pullRun(cmd *cobra.Command, args []string) {
 		pullTo = args[0]
 		if len(args) == 1 {
 			if transport == "" {
-				pullTo = uri.GetName("oras://" + pullFrom)
+				pullTo = uri.GetName("library://" + pullFrom)
```

Comment:
@DrDaveD @gmkurtzer do we really want `library://` to be default ?

Reply:
That default was removed by a later commit